src/clib/pio_rearrange.c

```diff
@@ -680,28 +682,13 @@ int compute_counts(iosystem_desc_t *ios, io_desc_t *iodesc, int maplen,
     }
 
     /* s2rindex is the list of indeces on each compute task */
```

Comment:
The previous blocks were the setup for the swapm call at line 685.   Here we are sending the mapping from the index on the compute task to the index on the io task.